fabric-data-generation-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/datagen/client/DataGeneratorClientTestEntrypoint.java

```diff
@@ -88,4 +91,22 @@ public void generateItemModels(ItemModelGenerator itemModelGenerator) {
 			//itemModelGenerator.register(item, Models.SLAB);
 		}
 	}
+
+	private static class TestSoundsProvider extends FabricSoundsProvider {
+		private TestSoundsProvider(DataOutput output, CompletableFuture<WrapperLookup> registryLookupFuture) {
+			super(output, registryLookupFuture);
+		}
+
+		@Override
+		public String getName() {
+			return "Test Sound Events";
+		}
+
+		@Override
+		protected void configure(SoundExporter exporter) {
+			exporter.add(DataGeneratorTestContent.TEST_SOUND, SoundTypeBuilder.of(DataGeneratorTestContent.TEST_SOUND)
```

Comment:
Expanding this to test both the "string" codec and maybe setting all of the values to a custom value would be easy thing to do.